### PR TITLE
add wishlist and cart Functionality with api

### PR DESCRIPTION
**Pull request by archit7448**
# What to check 

- [x] Reducer.jsx    

- [x] cart.jsx 

- [x] wishlist.jsx

### CODE REVIEW

**Review by Megha-Joshi — APPROVED** [review]
Good work! Names given to classes and variables are good.

**Review by Shubhamkr2610 — APPROVED** [review]
code is good and readable .Everything looks good to me.

File: src/Components/card-cart/card-cart.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      const { _id, title, produced, price, quantity, image, ratings } =
+        products;
+      return (
+        <div className="card-wrapper  card-cart" key={_id}>
+          <img src={image} alt={title} />
+          <div className="content-wrapper">
+            <h1 className="card-heading-main">{title}</h1>
+            <h2 className="card-heading-two">{produced}</h2>
+            <h2 className="card-price">₹ {price}</h2>
+            <h2 className="card-ratings"> ratings:{ratings}</h2>
```
Comment (iRohitGaur) [inline src/Components/card-cart/card-cart.jsx:18]: suggestion: too many h2 here. check for semantics

**Review by iRohitGaur — APPROVED** [review]
suggestion: please provide screenshots or preview to get a better idea of UI as well.

Good to merge with changes 👍🏻

File: src/Components/card-cart/card-cart.jsx (hunk 1 above)
Comment (archit7448) [reply src/Components/card-cart/card-cart.jsx:18]: I will correct in the refactor

**Pull request merged**